### PR TITLE
Add tables endpoint

### PR DESCRIPTION
**Pull request by ryanmitchell**
Endpoint + docs in new API format.

### CODE REVIEW

**Comment by sampoyigi** [comment]
Adding a new endpoint is amazingly clean and simple 🤯. Love it!!!

**Comment by ryanmitchell** [comment]
Most of the work is writing the docs :)

**Comment by sampoyigi** [comment]
Lol, we could go extreme and integrate something like Swagger to generate the docs 🤷‍♂️

**Comment by ryanmitchell** [comment]
Nah its good - we've done all the main ones so its just filling gaps from now.
I'll work on the new doc responses and rules this week.

**Comment by sampoyigi** [comment]
Hehehe, I wasn't serious about that. There's only so much we can do 😄

File: Extension.php [hunk 1]
```diff
@@ -140,6 +140,16 @@ public function registerApiResources()
                     'destroy:admin',
                 ],
             ],
+            'tables' => [
+                'controller' => \Igniter\Api\ApiResources\Tables::class,
+                'name' => 'Tables',
+                'description' => 'An API resource for tables',
+                'authorization' => [
+                    'index:users', 'show:users',
```
Comment (sampoyigi) [inline Extension.php:148]: All actions should be limited to just the admin.

**Review by sampoyigi — CHANGES_REQUESTED** [review]
Last one ;)

File: Extension.php (hunk 1 above)
Comment (ryanmitchell) [reply Extension.php]: Done :)

**Review by sampoyigi — APPROVED** [review]

**Pull request merged**